### ISSUE TITLE
portlist needs to collect IPv6 listening sockets as well as IPv4.

### ISSUE THREAD

**Issue by danderson** [tailscale/tailscale#1443]
On some platforms, portlist doesn't collect v6 listening sockets (at least linux). We need to collect them, now that IPv6 works. Also, on linux under some circumstances, binding to the unspecified address results in a v6-only bind (as reported by netstat, and seen by a user on twitter). So, by skipping v6, we miss out on some dual-stacked listening sockets.

**Comment by danderson** [comment]
Trivial repro:

```
package main

import "net/http"

func main() {
	http.ListenAndServe(":8081", nil)
}
```

`go run` that, and observe that in `netstat -tnlp`, the socket is reported only as tcp6, not tcp.

**Comment by danderson** [comment]
Getting this fixed in 1.6.0. In the meantime, a usable workaround should be to bind to `0.0.0.0:8081` explicitly, rather than just `:8081`.
